test/quantization/test_quantize_script.py

```diff
@@ -1233,6 +1233,98 @@ def _test_op_impl(self, module, data, quantized_op, tracing=False, debug=False,
 
         return models[False]
 
+    @skipIfNoFBGEMM
+    def test_quantized_linear(self):
```

Comment:
could you merge all the tests in one function? and add tests for tracing as well?

Reply:
Is there an example of a tracing tests that I could base on?

Reply:
for example: https://github.com/pytorch/pytorch/blob/master/test/quantization/test_quantize_script.py#L1237